core/json_schemas/src/schemas/parameters_values.yaml

```diff
@@ -0,0 +1,8 @@
+---
+oneOf:
+  - type: "null"
+  - type: "object"
+    additionalProperties: false
+    patternProperties:
+      "":
```

Comment:
lol: this is _so_ weird, I like it (although it could have been `.+` for clarity)

Reply:
Well I didn't even blink about it, but you are right, I changed it. Thanks